src/genericValidator/genericValidator.java

```diff
@@ -0,0 +1,36 @@
+package genericValidator;
+
+import genericValidator.processors.*;
```

Comment:
import genericValidator.processors.*; 
 "*" is imported all classes genericValidator.processors package.
 But it is not a good idea to use this version. Please write all imported classes manually.